vanilla-framework/vanilla-framework
# Update label to reflect input

## Done

Updated input date from 'Date' to 'Text' so example displays correctly.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- View https://docs.vanillaframework.io/en/base/forms and look at first example
- Check that input now reads 'text'

## Details

Fixes https://github.com/canonical-websites/vanillaframework.io/issues/191
